src/images/logos/product/brainly-logo-math-solver.svg

```diff
@@ -0,0 +1,5 @@
+<svg viewBox="0 0 80 80" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@domidomi the logo names are slightly different in the figma file and from where we left off yesterday. Could you please contact Olga and David to confirm what happened?

Reply:
@coderitual Olga is going to adjust filenames in Figma

Reply:
@coderitual seems that we should remove "brainly" from logos file names though!

Reply:
thanks for checking @domidomi